rfcs/text/0000-login-fields.md

```diff
@@ -0,0 +1,152 @@
+# 0000: Login Fields RFC
+
+Stage: 1
```

Comment:
@mr1716 please could you change the text `Stage: 1` to `Stage: 0`

We need to start the RFC process with a `Stage: 0` RFC